### PR TITLE
fix(DB/Commands): align core and DB

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
- Align GM Commands between Core and DB

I know that the security level of some commands is not ideal yet, but at least this PR makes it consistent.
We can change the security of certain commands later, and every time we do it we MUST update the wiki too.

### CODE REVIEW

File: src/server/scripts/Commands/cs_modify.cpp [hunk 1]
```diff
@@ -58,13 +58,13 @@ class modify_commandscript : public CommandScript
             { "standstate",     SEC_GAMEMASTER,      false, &HandleModifyStandStateCommand,    "" },
             { "phase",          SEC_GAMEMASTER,      false, &HandleModifyPhaseCommand,         "" },
             { "gender",         SEC_GAMEMASTER,      false, &HandleModifyGenderCommand,        "" },
-            { "speed",          SEC_GAMEMASTER,      false, nullptr,                           "", modifyspeedCommandTable }
+            { "speed",          SEC_GAMEMASTER,      false, nullptr,                    "", modifyspeedCommandTable }
```
Comment (Kitzunu) [inline src/server/scripts/Commands/cs_modify.cpp:61]: indent

File: src/server/scripts/Commands/cs_npc.cpp [hunk 2]
```diff
@@ -162,14 +162,14 @@ class npc_commandscript : public CommandScript
...
+            { "playemote",      SEC_GAMEMASTER,  false, &HandleNpcPlayEmoteCommand,         "" },
             { "say",            SEC_GAMEMASTER,     false, &HandleNpcSayCommand,               "" },
             { "textemote",      SEC_GAMEMASTER,     false, &HandleNpcTextEmoteCommand,         "" },
             { "whisper",        SEC_GAMEMASTER,     false, &HandleNpcWhisperCommand,           "" },
             { "yell",           SEC_GAMEMASTER,     false, &HandleNpcYellCommand,              "" },
             { "tame",           SEC_GAMEMASTER,     false, &HandleNpcTameCommand,              "" },
-            { "add",            SEC_GAMEMASTER,     false, nullptr,                            "", npcAddCommandTable },
-            { "delete",         SEC_GAMEMASTER,     false, nullptr,                            "", npcDeleteCommandTable },
+            { "add",            SEC_ADMINISTRATOR,     false, nullptr,                            "", npcAddCommandTable },
+            { "delete",         SEC_ADMINISTRATOR,     false, nullptr,                            "", npcDeleteCommandTable },
```
Comment (Kitzunu) [inline src/server/scripts/Commands/cs_npc.cpp:172]: indent

File: src/server/scripts/Commands/cs_modify.cpp (hunk 1 above)
Comment (Kitzunu) [reply src/server/scripts/Commands/cs_modify.cpp:61]: of "", modifyspeedCommandTable

File: src/server/scripts/Commands/cs_npc.cpp (hunk 2 above)
Comment (Kitzunu) [reply src/server/scripts/Commands/cs_npc.cpp:172]: of "", npcAddCommandTable "", npcDeleteCommandTable

File: src/server/scripts/Commands/cs_modify.cpp (hunk 1 above)
Comment (FrancescoBorzi) [reply src/server/scripts/Commands/cs_modify.cpp]: fixed, thanks

File: src/server/scripts/Commands/cs_npc.cpp (hunk 2 above)
Comment (FrancescoBorzi) [reply src/server/scripts/Commands/cs_npc.cpp]: fixed, thanks

**Comment by pklloveyou** [comment]
TEsted

**Review by Helias — APPROVED** [review]

**Comment by Voxstrasza** [comment]
@FrancescoBorzi had this error in worldserver
```
Table `command` have non-existing command 'bank', skip.
```

**Comment by FrancescoBorzi** [comment]
@Voxstrasza my bad, thanks. The command is actually `character check bank` and we already had it

**Comment by FrancescoBorzi** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/4515

**Pull request merged**